test/chisel2.py

```diff
@@ -32,8 +35,7 @@
 logger = logging.getLogger()
 logger.setLevel(int(os.getenv('LOGLEVEL', 0)))
 
-DIRECTORY = os.getenv('DIRECTORY', 'http://localhost:14000/dir')
-#DIRECTORY = os.getenv('DIRECTORY', 'http://localhost:4000/directory')
+DIRECTORY = os.getenv('DIRECTORY', 'http://localhost:4000/directory')
```

Comment:
I just remembered the WFE2 listens on [port 4001](https://github.com/letsencrypt/boulder/blob/master/test/config/wfe2.json#L3) not 4000. Sorry! Should have thought of that with the first review.